### ISSUE TITLE
Guide how to create simple Digital Twin with APIs

### ISSUE THREAD

**Issue by kyyberi** [PlatformOfTrust/pot-websites#94]
**Is your feature request related to a problem? Please describe.**
As a developer I find it hard to understand how Identity API is used to create Digital Twins. 

**Describe the solution you'd like**
Simple guide which shows how to use Identity API to create minimal Digital Twin containing:
* Building
* Floor 
* One room in floor
* One sensor in the above room. 

Code level step by step guide how to do the above. Code examples as cURL examples with necessary comments. Link to specific points in API Documentation in each step to offer more details.  

## Acceptance criteria

- [ ] Guide is written in markdown format in https://github.com/PlatformOfTrust/pot-websites/tree/master/developers-site/src/pages/md-pages subfolder `04-30-2019-simple-digital-twin-with-apis
- [ ] Guide contains step by step instructions with cURL code examples
- [ ] Each step is explained eg what happens
- [ ] Resulted Digital Twin can be retrieved with Identity API and sample code for that is provided too.

**Comment by kyyberi** [comment]
Meeting with @karihovi-vg on Wedn 15th about this.

**Comment by kyyberi** [comment]
Need the ontologies for the elements which we use in API calls

**Comment by kyyberi** [comment]
Floor has been defined in the standards: https://standards.oftrust.net/class-potjsonldfloor.html

**Comment by kyyberi** [comment]
Creating a building

```
curl -X POST https://api-sandbox.oftrust.net/identities/v1/ \
-H "Content-Type: application/json" \
-H "Authorization: Bearer REPLACE_WITH_YOUR_TOKEN" \
-d '{
    "context": "http://platformoftrust.github.io/standards/contexts/identity-building.jsonld",
    "type": "Building",
    "data": {
        "name": "Platform of Trust HQ",
        "description": "Platform of Trust company headquarters in Tampere."
    }
}'

```

In the above example I did cut some corners since the example in API docs has some redundancy in providing the data. I removed the name part from content root level to data. Still that "type" attribute seems possibly also redundant.

Would that above example be doable, would it work, does it make sense? :) @soderluk

**Comment by soderluk** [comment]
@kyyberi: The `name` attribute MUST be in the root level, as it's a defining attribute for the identity. Also, please use the correct URL for the contexts: `https://standards.oftrust.net/contexts/identity-building.jsonld`

**Comment by kyyberi** [comment]
ok. I do not know what is mandatory in the data model and what is not. Do I need to provide data to all of them for example `netFloorArea`, `stairwayAmount`and so on?

**Comment by soderluk** [comment]
You see it from the vocabulary: `https://standards.oftrust.net/vocabularies/building.jsonld`
For a building, at the moment, none of the additional `data` attributes are mandatory.

**Comment by kyyberi** [comment]
So this would be valid? 

```
curl -X POST https://api-sandbox.oftrust.net/identities/v1/ \
-H "Content-Type: application/json" \
-H "Authorization: Bearer REPLACE_WITH_YOUR_TOKEN" \
-d '{
    "context": "https://standards.oftrust.net/contexts/identity-building.jsonld",
    "type": "Building",
    "name": "Platform of Trust HQ",
    "data": {
        "description": "Platform of Trust company headquarters in Tampere."
    }
}'
```

In the `context`: 
<img width="747" alt="Näyttökuva 2019-5-16 kello 14 33 42" src="https://user-images.githubusercontent.com/1927158/57850860-b2a7c180-77e7-11e9-808d-92f77a0fbe07.png">

the `description` has definition `"@nest": "pot:data"`. Does that tell me that it should go under `data`?

**Comment by soderluk** [comment]
@kyyberi: You are correct. `@nest` tells that the value of description should be nested in the data object (the `@nest` shouldn't include the prefix, which is there now, but that will be fixed)
This looks like a valid request.

**Comment by kyyberi** [comment]
Excellent! Thanks @soderluk

**Comment by kyyberi** [comment]
Before linking entities those need to be created. For example before floor can be attached to building, both has to be created.

**Comment by kyyberi** [comment]
I cant find the `Device` JSON-LD in the contexts

<img width="1336" alt="Näyttökuva 2019-5-17 kello 9 08 21" src="https://user-images.githubusercontent.com/1927158/57906703-83df2900-7883-11e9-8707-77bcff43a354.png">

**Comment by kyyberi** [comment]
I will write the raw version of the guide in separate branch to github as far as possible. Nevertheless this issue can not be solved until the Device.jsonld is provided by the ontology team. Thus blocked tag is added.

**Comment by kyyberi** [comment]
Skeleton for the guide in topic level created in branch https://github.com/PlatformOfTrust/pot-websites/tree/feature/digital-twin-guide

**Comment by kyyberi** [comment]
If the Device JSON-LD is not ready 10.6.2019 we will do the #114 without the last step which is to add a device.
